modules/account-lib/src/coin/algo/utils.ts

```diff
@@ -29,6 +31,26 @@ function allHexChars(maybe: string): boolean {
   return maybe.match(/^[0-9a-f]+$/i) !== null;
 }
 
+function arrayEqual(a, b) {
+  if (a.length !== b.length) {
+    return false;
+  }
+  return a.every((val, i) => val === b[i]);
+}
+
+/**
+ * ConcatArrays takes two array and returns a joint array of both
```

Comment:
Seems to be more generic util. Please check if there a more generic utils file.

Reply:
https://dustinpfister.github.io/2018/08/02/lodash_concat/

Reply:
+1